src/java/fr/paris/lutece/plugins/blog/web/BlogJspBean.java

```diff
@@ -530,7 +530,7 @@ public String doAddTag( HttpServletRequest request )
                 return JsonUtil.buildJsonResponse( new JsonResponse( RESPONSE_BLOG_LOCKED ) );
             }
 
-        if ( RBACService.isAuthorized( Tag.PROPERTY_RESOURCE_TYPE, strIdTag, Tag.PERMISSION_CREATE, getUser( ) ) )
+        if ( RBACService.isAuthorized( Tag.PROPERTY_RESOURCE_TYPE, strIdTag, Tag.PERMISSION_CREATE, getUser( ) == null ? AdminUserService.getAdminUser( request ) : getUser()) )
```

Comment:
What is this fix for?

Reply:
these modifications take into account the evolution requested in the ticket 99710 : improvement of the creation of tickets on O2T